build_runner/lib/src/server/server.dart

```diff
@@ -110,6 +119,34 @@ class ServeHandler implements BuildState {
   }
 }
 
+class WebSocketHandler {
```

Comment:
This is pretty specific to LiveReload right now, we might make it more generic later but for now lets rename it to `LiveReloadWebSocketHandler` - or maybe `BuildUpdatesWebSocketHandler`.